embeds/linux_desktop.go

```diff
@@ -0,0 +1,79 @@
+package embeds
+
+import (
+	"bytes"
+	_ "embed"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/anywherelan/ts-dns/util/lineread"
+
+	"github.com/anywherelan/awl/config"
+)
+
+//go:embed .desktop
+var desktopFile []byte
+
+const (
+	desktopFileMod  = 0764 // rwxrw-r--
+	desktopFileName = "awl.desktop"
+	desktopFileDir  = ".local/share/applications"
+)
+
+func EmbedDesktopFile(iconPath string) error {
```

Comment:
Please add condition, that we don't embed desktop file if `config.IsDevVersion()`